From b8b1ea1caa81c95ade0272387b05a035658eed78 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Thu, 3 Oct 2024 13:12:35 +0200
Subject: [PATCH 688/954] fix(fdts): set flexgen 58 to HSE_KER for
 STM32MP257F-DK
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Flexgen 58 must set with HSE_KER for low power management

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 7601f030edfc3490d2f4a1ba9b7573016b1aff2d fix(fdts): set flexgen 58 to HSE_KER for STM32MP257F-DK]
Change-Id: I66d8c10c316913265de62726660a9e9e139db1ba
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/408814
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi b/fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
index e3ebc8275e..837abb43f3 100644
--- a/fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
+++ b/fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
@@ -50,7 +50,7 @@
 		FLEXGEN_CFG(48, XBAR_SRC_PLL5, 0, 3)
 		FLEXGEN_CFG(51, XBAR_SRC_PLL4, 0, 5)
 		FLEXGEN_CFG(52, XBAR_SRC_PLL4, 0, 5)
-		FLEXGEN_CFG(58, XBAR_SRC_HSE, 0, 1)
+		FLEXGEN_CFG(58, XBAR_SRC_HSE_KER, 0, 1)
 		FLEXGEN_CFG(63, XBAR_SRC_PLL4, 0, 2)
 	>;
 
-- 
2.53.0

